lib/players/human.rb

```diff
@@ -0,0 +1,15 @@
+class Player::Human < Player 
+
+  attr_accessor :board
+
+  def move(board)
+    puts "where on the board would you like to go? Choose a number from 1-9"
+    input = gets.strip
+  end 
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  puts "where on the board would you like to go? Choose a number from 1-9"
  gets.strip
end
```